scripts/link_bb.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+from os import path, makedirs
+from subprocess import run
+
+from qcore import simulation_structure
+from qcore import constants
+from shared_workflow.shared_automated_workflow import add_to_queue
+
+
+def arg_parser():
```

Comment:
Function name is too close to the module name. I recommend load_args inline with other repo scripts